components/Icon.tsx

```diff
@@ -3,15 +3,27 @@ import clsx from "clsx";
 type Props = {
   type: "home" | "search";
   className?: string;
-  size?: string;
+  size?: "small" | "medium";
 };
 
 export default function Icon({ type, className, size }: Props) {
```

Comment:
you'll probably want a default size

```suggestion
export default function Icon({ type, className, size = 'medium' }: Props) {
```